### PR TITLE
Research UI Improvements

### PR DESCRIPTION
**Pull request by ghost**
Changes to be committed:
	modified:   code/modules/research/rdconsole.dm

## About The Pull Request
I've changed the 'Normal View' On the research menu so it shows all available items on all sections.
I was hoping to do some other changes such as having non-researched items show up as read in their respective building machines.
I give up though. The Ui in this is pretty rough.

Before:
image

After:
uAtDVaHpT3

## Why It's Good For The Game

I've been going one by one through all the research nodes and checking what items they contain manually, this makes it much easier to see what you want to research along with what you have researched.

## Changelog
:cl:
tweak: Modified Normal View for RD computer.
/:cl:

### CODE REVIEW

**Comment by TeomanTheGreat** [comment]
Looks like someone left the changelog at default

**Comment by ghost** [comment]
> Looks like someone left the changelog at default

Fug

**Comment by Mick1299** [comment]
Should the expert view be replaced with the current normal view then?

**Comment by ghost** [comment]
> Should the expert view be replaced with the current normal view then?

Hm I don't know - I think with this people would only use either normal view or list view.

File: code/modules/research/rdconsole.dm [hunk 1]
```diff
@@ -698,7 +699,7 @@ Nothing else in the console has ID requirements.
 		display_name = "<A href='?src=[REF(src)];view_node=[node.id];back_screen=[screen]'>[display_name]</A>"
 	l += "<div class='statusDisplay technode'><b>[display_name]</b> [RDSCREEN_NOBREAK]"
 	if(minimal)
-		l += "<br>[node.description]"
+		l += ""
```
Comment (qwertyquerty) [inline code/modules/research/rdconsole.dm:702]: What's the point of this

File: code/modules/research/rdconsole.dm [hunk 2]
```diff
@@ -709,11 +710,11 @@ Nothing else in the console has ID requirements.
 				l += "<BR><span class='linkOff'>[node.price_display(stored_research)]</span>"  // gray - too expensive
 		else
 			l += "<BR><span class='linkOff bad'>[node.price_display(stored_research)]</span>"  // red - missing prereqs
-		if(ui_mode == RDCONSOLE_UI_MODE_NORMAL)
-			l += "[node.description]"
-			for(var/i in node.design_ids)
-				var/datum/design/D = SSresearch.techweb_design_by_id(i)
-				l += "<span data-tooltip='[D.name]' onclick='location=\"?src=[REF(src)];view_design=[i];back_screen=[screen]\"'>[D.icon_html(usr)]</span>[RDSCREEN_NOBREAK]"
+	if(ui_mode == RDCONSOLE_UI_MODE_NORMAL)
```
Comment (qwertyquerty) [inline code/modules/research/rdconsole.dm:713]: Why is this moved out of the else?

Comment (ghost) [reply code/modules/research/rdconsole.dm:713]: Because I want the check to occur regardless if it's in minimal mode.

File: code/modules/research/rdconsole.dm (hunk 1 above)
Comment (ghost) [reply code/modules/research/rdconsole.dm:702]: Because the list expects an entry regardless.

**Comment by Axanthic** [comment]
Closing - Was PermaBanned

**Comment by Flakkyboo** [comment]
Somebody steal the code. 😎

**Review by ghost — COMMENTED** [review]
> Somebody steal the code. 😎

shhh it never happened I have no clue what you're talking about
